nextflow.config

```diff
@@ -25,6 +25,7 @@ params {
 
     // predict epitopes
     pred_method = 'syfpeithi'
+    allow_all_peptide_lengths = false
```

Comment:
not sure, maybe this could be a bit  more descriptive to avoid misinterpretations and prevent users from setting this to true  (because it sounds like a param which should be true :D ) ?
e.g. `allow_inconsistent_pep_lengths` ?

Reply:
Thats better for sure! :)